PsevdoTwwiterApp/src/main/webapp/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="resources/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <title>PsevdoTwitter</title>
+
+   <script type="text/javascript" src="resources/js/script.js"></script>
+</head>
+<body>
+  
+  <button class="Log">Log out</button>
+  <h1>Valery</h1>
+ 
+   
+ <form action="/action_page.php">
+    <p class="filter">Filter</p>
+  <label for="Name">Name:</label><br>
```

Comment:
Attribute `for` in the label relates to `id` in input, not `name`. So, here you need `for="name"`